src/Rest/Helpers/Channel/AllowedMentionsBuilder.php

```diff
@@ -4,11 +4,15 @@
 
 namespace Exan\Fenrir\Rest\Helpers\Channel;
 
+use Exan\Fenrir\Rest\Helpers\GetNew;
+
 /**
  * @see https://discord.com/developers/docs/resources/channel#allowed-mentions-object
  */
 class AllowedMentionsBuilder
 {
+    use GetNew;
+
     private array $data = [
```

Comment:
```suggestion
    /**
     * @var array[] $data
     */
    private array $data = [
```
Added php type doc

Reply:
There's also an optional entry for `replied_user` which is a `bool`, so the doc is incorrect.

I don't think its a good idea to add docblocks for `$data` of builders, the exact structure can't be specified. Creates more confusion then it clears up IMO